Mathlib/Topology/MetricSpace/Polish.lean

```diff
@@ -76,9 +76,11 @@ class UpgradedPolishSpace (α : Type*) extends MetricSpace α, SecondCountableTo
   CompleteSpace α
 #align upgraded_polish_space UpgradedPolishSpace
 
-instance (priority := 100) polishSpace_of_complete_second_countable [m : MetricSpace α]
-    [SecondCountableTopology α] [h' : CompleteSpace α] : PolishSpace α where
-  complete := ⟨m, rfl, h'⟩
+instance (priority := 100) polishSpace_of_complete_second_countable [UniformSpace α]
```

Comment:
could you rename the instance to something more sensible? I don't think we should add deprecated versions for instances, as they will just make instance search slower.